lib/inspec/waiver_file_reader.rb

```diff
@@ -53,8 +71,10 @@ def self.validate_headers(headers, json_yaml = false)
     end
 
     def self.validate_json_yaml(data)
+      return if data.nil?
+
       headers = []
-      data.each_value do |value|
+      data&.each_value do |value|
```

Comment:
since you already check for nil, you don't need this safe check here

Reply:
Just a safe check